lib/plugins/ray_trace.js

```diff
@@ -23,4 +24,36 @@ module.exports = (bot) => {
 
     return bot.world.raycast(eyePosition, viewDirection, maxDistance, matcher)
   }
+
+  bot.playerAtCursor = (maxDistance = 3.5) => {
+    const players = Object.keys(bot.players).map((key) => bot.players[key])
+      .filter((player) => {
+        return player.username !== bot.username && player.entity &&
+          Math.abs(player.entity.position.x - bot.entity.position.x) <= maxDistance &&
```

Comment:
Position is a Vec3. You can use .distanceTo() and distanceSquared() on another Vec3.

You can also use Object.values when you want to get the values of an object, or Object.entries when you want an array of [key, value].